### PR TITLE
doc: relnotes: Bluetooth release notes for 2.0

### PR DESCRIPTION
**Pull request by carlescufi**
Add a section containing the most relevant Bluetooth changes for 2.0.

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

File: doc/releases/release-notes-2.0.rst [hunk 1]
```diff
@@ -339,7 +339,53 @@ Networking
 Bluetooth
 *********
 
-* TBD
+* Host:
+
+  * GATT: Added support for dabase hashes
```
Comment (jhedberg) [inline doc/releases/release-notes-2.0.rst]: database

File: doc/releases/release-notes-2.0.rst [hunk 2]
```diff
@@ -339,7 +339,53 @@ Networking
...
+  * GATT: Added support for Ready Using Characteristic UUID
+  * GATT: Added support for static services
+  * GATT: Added support for disabling the dynamic database
+  * GATT: Added support for notifying indicating by UUID
```
Comment (ioannisg) [inline doc/releases/release-notes-2.0.rst]: the above is not clear to me

File: doc/releases/release-notes-2.0.rst [hunk 3]
```diff
@@ -339,7 +339,53 @@ Networking
...
+  * Nordic: Added support for Controller-based privacy
+  * Nordic: Decorrelated address generation from resolution
+  * Nordic: Added support for fast encryption setup
+  * Nordic: Added support for RSSI
+  * Nordic: Added support for low-latency ULL processing of messages
+  * Nordic: Added support for nRF52811
+  * Nordic: Added support for PA/LNA on Port 1 GPIO pins
+  * Nordic: Added support for radio event abort
+  * BLE qualification up to the 5.1 specification
+  * Multiple bug fixes
```
Comment (ioannisg) [inline doc/releases/release-notes-2.0.rst]: Several

File: doc/releases/release-notes-2.0.rst [hunk 4]
```diff
@@ -339,7 +339,53 @@ Networking
...
+  * Added support for out-of-tree user-defined commands and events
+  * Added support for Zephyr Vendor Specific Commands
+  * Added support for user-defined protocols
+  * Converted several control procedures to be queuable
+  * Nordic: Added support for Controller-based privacy
+  * Nordic: Decorrelated address generation from resolution
+  * Nordic: Added support for fast encryption setup
+  * Nordic: Added support for RSSI
+  * Nordic: Added support for low-latency ULL processing of messages
+  * Nordic: Added support for nRF52811
```
Comment (ioannisg) [inline doc/releases/release-notes-2.0.rst]: That is confusing with the actual Board, SOC support

File: doc/releases/release-notes-2.0.rst (hunk 1 above)
Comment (carlescufi) [reply doc/releases/release-notes-2.0.rst]: Fixed

File: doc/releases/release-notes-2.0.rst (hunk 2 above)
Comment (carlescufi) [reply doc/releases/release-notes-2.0.rst]: Fixed

File: doc/releases/release-notes-2.0.rst (hunk 3 above)
Comment (carlescufi) [reply doc/releases/release-notes-2.0.rst]: Fixed

File: doc/releases/release-notes-2.0.rst (hunk 4 above)
Comment (carlescufi) [reply doc/releases/release-notes-2.0.rst]: Fixed

**Review by ioannisg — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**